index.html

```diff
@@ -701,8 +701,8 @@ <h2 id="constrainable-properties">
               <td><dfn class=export>suppressLocalAudioPlayback</dfn></td>
               <td>{{ConstrainBoolean}}</td>
               <td>
-                <p>As a setting, this value indicates whether or not the user
-                agent is applying <a>local audio playback suppression</a> to
+                <p>As a setting, this value indicates whether or not the constraint
+                for <a>local audio playback suppression</a> is applied to
```

Comment:
I'd prefer leaving the existing language alone here. As I recall, it was carefully chosen. cc @henbos 

[Local audio playback suppression](https://w3c.github.io/mediacapture-screen-share/#dfn-local-audio-playback-suppression) is already defined as a state in which _"the user agent SHOULD stop relaying audio to the local speakers"_ _(but importantly MAY NOT)._

This means we can be in this state, and we can reveal this state to JS, without revealing whether suppression is _actually_ happening.

This simultaneously satisfies the constrainable pattern, by having this state be the constrainable on/off property, while at the same allowing user agents the option to not comply in a manner that remains unobservable.

Reply:
I think @jan-ivar meant to @mention me here, not Henrik. I was involved with suppressLocalAudioPlayback, and @henbos was involved with the adjacent control [restrictOwnAudio](https://www.w3.org/TR/screen-capture/#dfn-restrictownaudio). (A most excellent control which I am hoping we'll be able to implement soon-ish, btw).

My concern with the PR as it stands is that it produces an apparent contradiction between these two sections:
1. suppressLocalAudioPlayback of type boolean [...] Indicates whether or not the user agent is applying local audio playback suppression to the source.
2. When local audio playback suppression is applied [...] suppression MUST NOT be observable to the captured document.

I'd prefer a formulation in no1 that is more immediately clear to readers who only check one part of the document. François, please consider the following:
> suppressLocalAudioPlayback of type boolean [...] Indicates whether or not the application instructed the user agent to apply local audio playback suppression to the source.

Or even:
> suppressLocalAudioPlayback of type boolean [...] Indicates whether or not the application instructed the user agent to apply local audio playback suppression to the source. This setting does not reveal to the application whether the user agent is actually applying local audio playback suppression, as this can be overridden by the user.

Reply:
I've applied your suggestion @eladalon1983.
@jan-ivar Does that still look good to you? If so, feel free to merge.